reference/configuration/assetic.rst

```diff
@@ -50,7 +50,7 @@ Full Default Configuration
                 some_filter:                 []
             workers:
                 # see https://github.com/symfony/AsseticBundle/pull/119
-                # Cache can also be busted via the framework.templating.assets_version
+                # Cache can also be busted via the framework.assets.version
```

Comment:
Well, that's no breaking change, but is a misconfiguration of the templating system.